website/lab/02/index.mdx

```diff
@@ -569,19 +569,48 @@ While the device initialization is specific to each hardware family, the pin ini
 ### Configure GPIO Input
 
 Using a pin as input is very similar to using it as output.
+<Tabs>
+<TabItem value="stm32u5" label="STM32 Nucleo-U545RE-Q" default>
 
 ```rust
-use gpio::{Input, Pull};
+use embassy_stm32::gpio::{Input, Pull};
 
+// initialize PXn
+//   - replace X with the port letter(A, B, C, D, E, etc.)
+//   - n with the pin number
+//   ... see the datasheet, e.g. PA5 
+// and activate its internal pull up resistor.
 let pin = Input::new(peripherals.PXn, Pull::Up);
 
 if pin.is_high() {
     // Do something if the pin value is HIGH (1)
 } else {
     // Do something if the pin value if LOW (0)
 }
-```
 
+</TabItem>
+
+<TabItem value="rp2350" label="Raspberry Pi Pico 1 / 2" default>
+
+```rust
+use embassy_rp::gpio::{Input, Pull};
+
+// initialize PXn
+//   - replace X with the port letter(A, B, C, D, E, etc.)
+//   - n with the pin number
+//   ... see the datasheet, e.g. PA5 
+// and activate its internal pull up resistor.
```

Comment:
Please us a similar comment with the one used for output pins, This is incorrect, the Pi has no ports for GPIOs.